appengine_flexible/mailjet/mailjet.go

```diff
@@ -49,20 +49,26 @@ func sendEmail(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	m := &mailjet.InfoSendMail{
-		FromEmail: fromEmail,
-		FromName:  "App Engine Mailjet sample",
-		Subject:   "Your email flight plan!",
-		TextPart:  "Dear passenger, welcome to Mailjet! May the delivery force be with you!",
-		HTMLPart:  "<h3>Dear passenger, welcome to Mailjet!</h3><br/>May the delivery force be with you!",
-		Recipients: []mailjet.Recipient{
-			{
-				Email: to,
-			},
-		},
-	}
+messagesInfo := []mailjet.InfoMessagesV31 {
+      mailjet.InfoMessagesV31{
+        From: &mailjet.RecipientV31{
+          Email: fromEmail,
+          Name: "Mailjet Pilot",
+        },
+        To: &mailjet.RecipientsV31{
+          mailjet.RecipientV31 {
+            Email: to,
+            Name: "passenger 1",
+          },
+        },
+        Subject: "Your email flight plan!",
+        TextPart: "Dear passenger, welcome to Mailjet! May the delivery force be with you!",
+        HTMLPart: "<h3>Dear passenger, welcome to Mailjet!</h3><br />May the delivery force be with you!",
+      },
+    }
 
-	resp, err := mailjetClient.SendMail(m)
+	messages := mailjet.MessagesV31{Info: messagesInfo }
```

Comment:
Please run gofmt. The space before the } shouldn't be there, and there are spaces used for indentation above.